.github/workflows/auto-updates.yaml

```diff
@@ -102,3 +104,51 @@ jobs:
         if: ${{ env.modified == 'true' }}
         run: |
           git push origin auto-update-readme-cli-ref:main
+
+  update-vendored-sources:
+    name: Update XS-Vendored-Sources-Rust on debian/control
+    runs-on: ubuntu-latest
+    container: 
+      image: ubuntu:23.04
```

Comment:
you’re right, I meant to use `ubuntu:rolling` rather. This will always use the last stable version (devel is always dangerous in case of some deps breaking)